src/plugins/discover/public/customizations/customization_types/flyout_customization.ts

```diff
@@ -21,10 +25,23 @@ export interface FlyoutActionItem {
   enabled: boolean;
 }
 
+export interface FlyoutContentActions {
+  setFilter?: DocViewFilterFn;
```

Comment:
```suggestion
  addFilter?: DocViewFilterFn;
```
I think `addFilter` would be more accurate here since it appends to the existing filters.